.github/workflows/reusable_go_lint_test.yml

```diff
@@ -198,3 +207,13 @@ jobs:
             echo "Uncommitted changes detected. Please run 'make mock-gen' before committing."
             exit 1
           fi
+  gosec:
+    runs-on: ubuntu-22.04
```

Comment:
Not related to your PR, but we should bump the ubuntu version to `24.04` for all the pipelines on this or a separate PR.